README.md

```diff
@@ -140,6 +144,17 @@ qr.modules.each do |row|
 end
 ```
 
+## Specifying QR code mode
+
+Sometimes you may want to specify the QR code mode explicitly. 
+
+It is done voia the `mode` option. Allowed values are: `number`, `alphanumeric`, `8bit_byte` and `kanji`.
```

Comment:
should it maybe be "via the mode"? Also we don't support kanji, so maybe leave it out for now ?